File: src/main/java/no/novari/flyt/egrunnerverv/gateway/slack/SlackConfig.java
package no.novari.flyt.egrunnerverv.gateway.slack;

import org.springframework.context.annotation.Bean;
import org.springframework.context.annotation.Configuration;
import org.springframework.web.reactive.function.client.WebClient;

@Configuration
public class SlackConfig {
    @Bean
    public WebClient slackWebClient(WebClient.Builder builder) {
        return builder.build();
    }
}

File: src/main/kotlin/no/novari/flyt/egrunnerverv/gateway/slack/SlackConfig.kt
package no.novari.flyt.egrunnerverv.gateway.slack

import org.springframework.context.annotation.Bean
import org.springframework.context.annotation.Configuration
import org.springframework.web.client.RestClient

@Configuration
class SlackConfig {
    @Bean
    fun slackRestClient(builder: RestClient.Builder): RestClient = builder.build()
}
